doc/api/esm.md

```diff
@@ -1153,22 +1180,12 @@ for looking up conditional mappings elsewhere or to modify the list when calling
 the default resolution logic.
 
 The [current set of Node.js default conditions][Conditional Exports] will always
-be in the `context.conditions` list passed to the hook. If the hook wants to
-ensure Node.js-compatible resolution logic, all items from this default
-condition list **must** be passed through to the `defaultResolve` function.
+be in the `context.conditions` list passed to the hook. When calling to a parent
+loader, Node.js-compatible resolution behavior is not guaranteed unless all
+default conditions are passed.
 
 ```js
-/**
- * @param {string} specifier
- * @param {object} context
- * @param {string} context.parentURL
- * @param {string[]} context.conditions
- * @param {function} defaultResolve
- * @returns {object} response
- * @returns {string} response.url
- */
-export async function resolve(specifier, context, defaultResolve) {
-  const { parentURL = null } = context;
+export async function resolve(specifier, parentURL, context, parentLoader) {
```

Comment:
Why is `parentURL` getting pulled out of `context`? Can't it stay in there?

Reply:
they are equally important in terms of this hook, I'm not sure why parentURL was stuffed into the context to begin with.

Reply:
Fair enough, but this change is unrelated to adding chaining support. Can these API redesigns please be part of their own PR?